main.js

```diff
@@ -0,0 +1,211 @@
+const { InstanceBase, Regex, runEntrypoint } = require('@companion-module/base')
+const getPresetDefinitions = require('./presets')
+
+
+class OSCInstance extends InstanceBase {
+	constructor(internal) {
+		super(internal)
+	}
+
+	async init(config) {
+		this.config = config
+		this.updateStatus('ok')
+		this.updateActions() // export actions
+		this.setPresetDefinitions(getPresetDefinitions(this))
+	}
+
+	async destroy() {
+		this.log('debug', 'destroy')
+	}
+
+	async configUpdated(config) {
+		this.config = config
+	}
+
+	getConfigFields() {
+		return [
+			{
+				type: 'textinput',
+				id: 'host',
+				label: 'Target IP',
+				width: 8,
+				regex: Regex.IP,
+			},
+			{
+				type: 'textinput',
+				id: 'port',
+				label: 'Target Port',
+				width: 4,
+				regex: Regex.PORT,
+			},
+		]
+	}
+
+	updateActions() {
+		const sendOscMessage = (path, args) => {
+			this.log('debug', `Sending OSC ${this.config.host}:${this.config.port} ${path}`)
+			this.log('debug', `Sending Args ${JSON.stringify(args)}`)
+			this.oscSend(this.config.host, this.config.port, path, args)
+		}
+		const actionDefinitions = {};
+
+		// Adicionando send_ppt
+		for (let i = 1; i <= 20; i++) {
```

Comment:
You still completely don't understand, what you are doing wrong and I really don't know how I can explain it in even more detail.
1. Do not make 20 different actions if there is only one parameter changing. Instead make this _**one**_ action where the changing parameter is an _**option**_.
2. There is completely no use in your 'COMANDO' in beinig a user editable textinput. Actually this makes all your actions the same action.